### PR TITLE
Add Contributing.md (fixes #268 for now)

### LINKED ISSUE #268: Contributing.md
- Github
- ...

### PR DESCRIPTION
**Pull request by carlobeltrame**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+# Contributing
+
+> Für eine deutsche Übersetzung siehe unten.
+
+eCamp is currently being redeveloped in this repository by a core developer team. This team consists of the original developers of eCamp v2, as well as a few other interested individuals. We are currently re-building the functionality of eCamp v2 in a more extensible and flexible way, before starting any new features.
+
+## Feature requests
+If you have some feature requests, please direct them to cosinus@gryfensee.ch, so we can build a list of requirements and wishes for future features. We are well aware of the need for offline usability and mobile friendly design. However, please note that it is still a long way until the program will be completed.
```
Comment (pmattmann) [inline CONTRIBUTING.md]: Change this to Github-Issues

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+## Helping out with development
+If you are a developer and wish to get involved in the project, please leave us a message at cosinus@gryfensee.ch. We will then get in touch with you once the project is ready for community development. Until then, feel free to set up the project on your local machine and make yourself familiar with it. Setup documentation can (soon) be found in our Wiki pages on GitHub.
+
+## Contribution process
```
Comment (pmattmann) [inline CONTRIBUTING.md]: Language is english

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+## Before submitting pull requests
+
+* Did cs-fixer run on all changed or new PHP files?
+* Did the test coverage stay the same or get higher across the project?
+* Are all passwords, credentials and local configuration removed from the code changes?
+* Do all changed files contain some important changes (as opposed to e.g. only whitespace changes)?
+* Is the fork up-to-date with the central repository and can your changes be merged automatically?
+* Did the travis CI build run through without test failures?
+
```
Comment (carlobeltrame) [inline CONTRIBUTING.md:79]: Language of all variables etc.

**Comment by carlobeltrame** [comment]
Add link to wiki

**Comment by carlobeltrame** [comment]
All requested changes implemented. @pmattmann please review

**Pull request merged**